sunpy/visualization/imageanimator.py

```diff
@@ -1002,12 +1002,29 @@ def _get_main_axes(self):
         return axes
 
     def _set_unit_in_axis(self, axes):
+        self._set_image_axis_units(unit_x_axis, unit_y_axis)
         if self.unit_x_axis is not None:
             axes.coords[2].set_format_unit(self.unit_x_axis)
-            axes.coords[2].set_ticks(exclude_overlapping=True)
+            axes.coords[2].set_ticks(spacing = 10*u.arcmin, exclude_overlapping=True)
```

Comment:
What if the units are not arcmins?

Reply:
Let the user decide the units ?

Reply:
Well for WCS, it should come from the header.